services/wfm/src/main/java/org/openkilda/wfm/topology/event/OFELinkBolt.java

```diff
@@ -403,6 +407,8 @@ private void dispatchMain(Tuple tuple, InfoMessage infoMessage) {
             passToTopologyEngine(tuple);
         } else if (data instanceof IslInfoData) {
             handleIslEvent(tuple, (IslInfoData) data, infoMessage.getCorrelationId());
+        } else if (data instanceof DiscoPacketSendingConfirmation) {
```

Comment:
DiscoPacketSendingConfirmation messages is much more frequent than other messages in this "dispatcher". It should be first in the "check" list.